department-of-veterans-affairs/caseflow
# Queue | Add judge checkout flow dropdown to CaseSnapshot, remove from JudgeReviewTaskTable

Resolves #5504

### Description
Adds the judge checkout flow dropdown to CaseSnapshot, removes it from JudgeReviewTaskTable.

### Acceptance Criteria 
- [ ] As Judge, dropdown options in Case Detail page should be "Ready for Dispatch" and "Assign OMO"
- [ ] Checkout flow dropdown not present in JudgeReviewTaskTable

### Testing Plan
1. Go to `/queue/` as `BVAAABSHIRE` / judge, navigate to detail page
2. Confirm Assign OMO takes you back to `/queue` with success message, Ready for Dispatch goes to Select Dispositions view
